neurovault/apps/statmaps/models.py

```diff
@@ -347,3 +351,33 @@ class Atlas(Image):
 
     class Meta:
         verbose_name_plural = "Atlases"
+
+
+class Similarity(models.Model):
```

Comment:
I am still not convinced we need this class in the production server. It will be a significant overhead without clear benefit. I am happy to be convinced otherwise.